pandas/tests/frame/methods/test_to_records.py

```diff
@@ -508,3 +508,11 @@ def test_to_records_datetimeindex_with_tz(self, tz):
 
         # both converted to UTC, so they are equal
         tm.assert_numpy_array_equal(result, expected)
+
+    def test_to_records_no_typeerror_in_repr(self):
```

Comment:
Could you move this to `pandas/tests/frame/test_repr_info.py`?

Reply:
ofc, I was not sure where to put the test exactly, so I will change its location.